book/hooks.md

```diff
@@ -239,7 +239,7 @@ The render array you want to change will be in the content variable which shows
 
 Usually fields such as field_image will be automatically rendered by the node template (unless you've tweaked it to display in some other template.)
 
-In your node\'s Twig template you would specify `{{ custom_variable }}` to have it display on every node.
+In your node\'s Twig template you would specify <code v-pre>{{ custom_variable }}</code> to have it display on every node.
```

Comment:
@maks-oleksyuk I'm curious about this code v-pre stuff. I see in the vitepress docs that we could use:
`This <span v-pre>{{ will be displayed as-is }}</span>`

Reply:
Yes, I think we can use this attribute with <span> as well, I tested it and it works, the only difference is that there is no highlighting as when using `